lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -205,6 +205,12 @@ void getAsmBlockArgumentNamesImpl(Operation *op, mlir::Region &region,
   }
 }
 
+static bool hasPortNamed(FModuleLike op, StringAttr name) {
```

Comment:
This method is no longer needed (dropped with IST update).  FWIW it appears to also be the only user of `hasSymNamed` but that doesn't mean it's not useful.

Reply:
Removed it.